lib/modules/easterEgg.js

```diff
@@ -23,11 +21,10 @@ module.go = () => {
 			notificationID: 'konami',
 		});
 		setTimeout(() => $baconBit.addClass('makeitrain'), 500);
-	};
+	});
 	konami.iphone.load = () => {}; // nix touch support; occasional false positives on touchscreen laptops
-	konami.load();
 };
 
 export function konamiActive() {
-	return konami && konami.almostThere;
+	return konami;
```

Comment:
Yes, I see the issue now. I've resurrected the old local konami code and added these changes. I have questions a specific line. Follow up incoming.